src/EnergyPlus/UtilityRoutines.hh

```diff
@@ -605,11 +605,13 @@ namespace UtilityRoutines {
     // For map, you'd only need the comparator
     struct case_insensitive_hasher
     {
+        using is_transparent = void;
         size_t operator()(std::string_view key) const noexcept;
     };
 
     struct case_insensitive_comparator
     {
+        using is_transparent = void;
```

Comment:
I was not aware of this C++14 feature.  Feels a bit magic at first, but I get it, and it's fine.

Reply:
Yes, it's magic.